src/components/Create/components/RecommendedTitle/RecommendedTitle.tsx

```diff
@@ -0,0 +1,27 @@
+import { Trans } from '@lingui/macro'
+import { ThemeContext } from 'contexts/themeContext'
+import { useContext } from 'react'
+
+export const RecommendedTitle = ({ text }: { text: string }) => {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  return (
+    <>
+      {text}{' '}
+      <span
+        style={{
+          backgroundColor: colors.background.action.secondary,
+          color: colors.text.action.primary,
+          borderRadius: '2.5rem',
+          textTransform: 'uppercase',
+          padding: '0.25rem 0.625rem',
+          position: 'relative',
+          fontSize: '12px',
+        }}
+      >
+        <Trans>Recommended</Trans>
+      </span>
```

Comment:
**suggestion**: WDYT about instead creating a component for this, and just using it where we need to in titles?

We have `src/components/Badge.tsx`. I think it can prolly be adapted to support the blue color.

Reply:
Done